Tools/machines/lumi-csc/lumi_warpx.profile.example

```diff
@@ -50,4 +57,4 @@ export CXX=$(which amdclang++)
 export FC=$(which amdflang)
 export CFLAGS="-I${ROCM_PATH}/include"
 export CXXFLAGS="-I${ROCM_PATH}/include -Wno-pass-failed"
-export LDFLAGS="-L${ROCM_PATH}/lib -lamdhip64"
+export LDFLAGS="-L${ROCM_PATH}/lib -lamdhip64 ${PE_MPICH_GTL_DIR_amd_gfx90a} -lmpi_gtl_hsa"
```

Comment:
Do they ask to add `-lmpi_gtl_hsa` in the LUMI documentation or did you find out that leaving this out leads to issues?

Reply:
I did not yet test this with WarpX, but I had issues starting ImpactX without it when GPU-aware MPI is enabled.